File: ruby/fibonacci.rb
#Printing n terms of a fibonacci series
puts "Enter the number"
n=gets
a=0
b=1
puts "The Fibonacci Series is"
puts a
puts b
for i in 2..Integer(n)-1 do
  c=a+b
  puts c
  a=b
  b=c
end

File: src/java/fibonacci.java
/*Fibonacci of n numbers */
import java.io.*;

class fib
{
 int f1=0,f2=1,f3;
  void sequence()
  {
   f3=f1+f2;
   f1=f2;
   f2=f3;
  } 
} 
class fibonacci
{
 public static void main(String args[])throws IOException
 {
  int i,n;
  DataInputStream dis=new DataInputStream(System.in);
  System.out.println("Enter n Value :");
  n=Integer.parseInt(dis.readLine());
  fib f=new fib();
  System.out.println("Fibonacci sequence is :");
  System.out.println(f.f1);
  System.out.println(f.f2);
  for(i=2;i<n;i++)
   {
  	f.sequence();
  	System.out.println(f.f3);
   }
  }
 } 